<?xml version="1.0"?>
<robot name="puzzle_base">
  <material name="block_wood">
    <color rgba="0.93 0.086 0.071 1" />
  </material>
  <link name="base_link">
    <visual>
      <origin xyz="0 0 0" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
      <geometry>
        <box size="0.4 0.52 0.05" />
      </geometry>
      <material name="block_wood"/>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
      <geometry>
        <box size="0.4 0.52 0.05" />
      </geometry>
    </collision>
    <inertial>
      <mass value="20.0" />
      <origin xyz="0 0 0" />
      <inertia ixx="1.0" ixy="0" ixz="0" iyy="1.0" iyz="0" izz="1.0" />
    </inertial>
  </link>
  <gazebo reference="base_link">
    <visual><material>
      <ambient>0.5 0.5 0.5 1</ambient>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <emmisive>0.2 0.2 0.2 1</emmisive>
    </material></visual>
  </gazebo>
  <!-- fixed to world -->
  <link name="world"/>
  <joint name="world_to_base_link" type="fixed">
    <parent link="world"/>
    <child  link="base_link"/>
  </joint>
</robot>
